app/views/job_invocations/_form.html.erb

```diff
@@ -74,6 +74,14 @@
                     <% end %>
 
                     <div class="advanced hidden">
+                      <%= job_template_fields.fields_for :provider_input_values do |provider_input_fields| %>
```

Comment:
Wouldn't it be more straight forward if we would have the namespaces here as well?
Just an idea, I don't insist, but seems like it :shrug:

Reply:
Well, it would add another level of nesting into the form so I am a bit hesitant to call it straightforward, but I can certainly add it if you prefer.

Reply:
no, I'm ok both ways, I just don't like going totally different path API vs UI, but this is minor difference for a good reason, so I can live with that :+1: